example.rb

```diff
@@ -1,7 +1,11 @@
 class Example
 
-  def rails_conf
+  def rails_conference
     puts 'RailsConf 2015!'
+    puts "in Atlanta"
+  end
+  def hello_world
+    puts 'Hello World!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.